Packs/Slack/Integrations/SlackV3/SlackV3.yml

```diff
@@ -101,6 +101,12 @@ configuration:
   name: enable_dm
   required: false
   type: 8
+- display: Common Channels
+  name: common_channels
+  type: 12
+  required: false
+  additionalinfo: For workspaces where a handful of channels are consistently being
```

Comment:
we will need to add in the readme that this may be needed if we get timeouts in {put here the relevant commands that use this}